src/appleseedmaya/typeids.h

```diff
@@ -64,7 +64,7 @@ enum AppleseedMayaTypeIds
     AsPlasticNodeTypeId             = 0x001279d7,   // 1210839
     AsThinFilmNodeTypeId            = 0x001279d8,   // 1210840
     AsIdManifoldNodeTypeId          = 0x001279d9,   // 1210841
-    AsVaryTextureNodeTypeId         = 0x001279da,   // 1210842
+    AsSwitchTextureNodeTypeId       = 0x001279da,   // 1210842
```

Comment:
We're not breaking compatibility here right? It's just a renaming?

Reply:
just renaming, this node is new.